.github/workflows/linux.yml

```diff
@@ -222,14 +222,14 @@ jobs:
 
       - name: Build Debian packages
         run: |
-          /usr/bin/python3.8 -m pip install -U pip
-          /usr/bin/python3.8 -m pip install -r ${OPENVINO_REPO}/src/bindings/python/wheel/requirements-dev.txt
+          /venv3.10/bin/python3 -m pip install -U pip
+          /venv3.10/bin/python3 -m pip install -r ${OPENVINO_REPO}/src/bindings/python/wheel/requirements-dev.txt
```

Comment:
do we need to prefix with `/venv3.10/bin/` ? is it possible to activate venv in `.bashrc` of docker image?

Reply:
There is already the Py3.11 being in `PATH` so that `python3` points to Py3.11 by default: 
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/1f7cb26d-0d61-4ecd-88f2-32c47340d3b0) so I had to resort to this approach.

We can do something like `alias python3.10=/venv3.10/bin/python3` but I think it is not necessary as it is used in one place only.